internal/backend/storage/gitfs/git.go

```diff
@@ -235,7 +235,7 @@ func (g *Git) defaultRemote(ctx context.Context, branch string) string {
 func (g *Git) defaultBranch(ctx context.Context) string {
 	out, _, err := g.captureCmd(ctx, "defaultBranch", "rev-parse", "--abbrev-ref", "HEAD")
 	if err != nil || string(out) == "" {
-		return "master"
+		return "main"
```

Comment:
Why change the default?

Reply:
See e.g. https://github.com/github/renaming for context.